readme/BREAKING_CHANGES_AMENDMENTS.markdown

```diff
@@ -6,7 +6,20 @@ so the correct message on **What** section should be
 
 modules/apps/document-library/document-library-repository-external-api/src/main/java/com/liferay/document/library/repository/external/ExtRepository.java
 
-modules/apps/document-library/document-library-repository-external-api/src/main/java/com/liferay/document/library/repository/external/ExtRepository.java now includes a new method generatePDFPreviews(long userId);
+```
+LPS-197315 add new param to the addExtRepositoryFileEntry method with the fileName of the file Entry
+
+# breaking
+
+## What modules/apps/document-library/document-library-repository-external-api/src/main/java/com/liferay/document/library/repository/external/ExtRepository.java
+
+com/liferay/document/library/repository/external/ExtRepository.java has changed the method ExtRepositoryFileEntry addExtRepositoryFileEntry(String extRepositoryParentFolderKey, String mimeType, String title, String description, String changeLog, InputStream inputStream) to ExtRepositoryFileEntry addExtRepositoryFileEntry(String extRepositoryParentFolderKey, String fileName, String mimeType, String title, String description, String changeLog, InputStream inputStream) adding the new parameter fileName.
```

Comment:
Hi @almonwork 

The old markdown was telling about the method `generatePDFPreviews` and the new about `addExtRepositoryFileEntry`

So I do not understand the change.
​
Have I missed something?